web/src/components/common/Header.tsx

```diff
@@ -27,15 +27,22 @@ export const Header = (): JSX.Element => {
     axios
       .get('version.txt')
       .then((response: AxiosResponse<string>) => {
-        const responseSubString = response.data.substring(
-          response.data.indexOf('tag:')
+        // get ref part from commit
+        const refSubString = response.data.substring(
+          response.data.lastIndexOf('('),
+          response.data.lastIndexOf(')') + 1
```

Comment:
What if the commit message is `fix(dialog): Fixed the dialog`? Then you would get `refSubString == "dialog"`

Reply:
Nope, because refSubString extracts from reverse only the refs (which is always included, inside parentheses and always at the end of the string).

Reply:
Right, you used `lastIndexOf`, then it would work, I suppose.